source/openapi-admin-v3.yaml

```diff
@@ -6851,7 +7056,7 @@ components:
             type: string
         collection_queryable_fields_names:
           type: object
-          description:  |-
+          description: |-
```

Comment:
My Sync config has a handful of fields that aren't represented here (`type`, `development_mode_enabled`, `service_name`, `database_name`). I notice there's still an outstanding comment from the last review where you suggested a separate ticket to look at ingest/sync config stuff. Does that ticket exist somewhere so we can capture these details?

Reply:
https://jira.mongodb.org/browse/DOCSP-36931